pipelines/azure-pipelines.remove-pod.yml

```diff
@@ -0,0 +1,69 @@
+parameters:
+  - name: env
+    type: string
+    default: dev
+    values:
+      - dev
+      - demo
+      - test
+      - ithc
+      - stg
+
+  - name: cluster
+    type: string
+    values:
+        - 00
+        - 01
+
+  - name: project
+    type: string
+    default: vh
+
+  - name: app_name
+    type: string
+    default: bookings-api
+
+  - name: pod_suffix
+    type: string
+    default: pr-396
```

Comment:
I would remove defaults here as it can be used by any app and should be forced to use

Reply:
I'm a-ok with removing the default but (maybe I'm being too hand-holdy) I thought it might be good to have a pre-populated box showing the format of the pod to be removed

Reply:
probably yer you are right.
if we remove the project and app_name defaults, then at least it won't accidently trigger on the wrong pod